drakrun/drakrun/playground.py

```diff
@@ -85,6 +102,8 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_value, exc_traceback):
         self.vm.destroy()
+        delete_vm_network(self.vm.vm_id, True, find_default_interface(), self._dns)
+        stop_dnsmasq(self.vm.vm_id)
```

Comment:
This will be handled by `graceful_exit`

Reply:
Oh okay, will remove it

Reply:
oh wait, graceful_exit will only `stop dnsmasq` right. what about the `vm_network`, will it also be cleared?

Reply:
Yes, dnsmasq will be stopped. delete_vm_newtork seems to be ok, though.

Reply:
Sure, will just remove `stop_dnsmasq` line